### ISSUE TITLE
Question about the reslute mAP.

### ISSUE THREAD

**Issue by ayumiymk** [longcw/faster_rcnn_pytorch#17]
Does anyone get the comparable mAP on VOC07 in the origin paper?

**Comment by zhanghang1989** [comment]
The author reported here:
https://github.com/longcw/faster_rcnn_pytorch#training-on-pascal-voc-2007

**Comment by ayumiymk** [comment]
But the author got a 0.661 mAP on VOC07 while the origin paper got a 0.699 mAP. 
The author said we may need to tune the loss function defined in faster_rcnn/faster_rcnn.py by ourself. So  what I want to know is the config to get a mAP comparable to the origin paper.

**Comment by acgtyrant** [comment]
I rewrite the project, and calculate its mAP, only 0.356.

**Comment by zchrissirhcz** [comment]
I just git cloned this project and train on my 1080Ti machine. The finally mAP@100000iteration is 65.6%.

**Comment by brisker** [comment]
@zchrissirhcz 
so what does the author mean by "tune the loss by ourselves" to get 0.699 mAP?

**Comment by zchrissirhcz** [comment]
@brisker In my opinion, I appreciate longcw's re-implementation. However, I'd like to use implementations with equal or higher accuracy than the paper. I suggest try this implementation:https://github.com/ruotianluo/pytorch-faster-rcnn

**Comment by brisker** [comment]
@zchrissirhcz 
https://github.com/ruotianluo/pytorch-faster-rcnn 
this implementation seems to be more consistent with rbg's original caffe version implementation.
But I like longcw's implementation more.
But still, I have no idea where to modify the loss function, as longcw have indicated. Do you know?
